plugins/kubectl-nifikop

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019 Orange
```

Comment:
I think this copyright needs tweaked. We should create lint rules to enforce that the license header is the same across all source files.

Reply:
I raised #173 to address all of the licensing later on. For now this is probably okay.